app/src/main/java/org/stepic/droid/ui/util/BottomNavigationUtil.java

```diff
@@ -0,0 +1,32 @@
+package org.stepic.droid.ui.util;
+
+import android.support.design.internal.BottomNavigationItemView;
+import android.support.design.internal.BottomNavigationMenuView;
+import android.support.design.widget.BottomNavigationView;
+import android.util.Log;
+
+import java.lang.reflect.Field;
+
+public class BottomNavigationUtil {
+    public static void disableMenuShiftMode(BottomNavigationView view) {
+        BottomNavigationMenuView menuView = (BottomNavigationMenuView) view.getChildAt(0);
+        try {
+            Field shiftingMode = menuView.getClass().getDeclaredField("mShiftingMode");
+            shiftingMode.setAccessible(true);
+            shiftingMode.setBoolean(menuView, false);
+            shiftingMode.setAccessible(false);
```

Comment:
Interesting how Google managed to fit 4 items in bottom bar in apps like [Playbook](https://play.google.com/store/apps/details?id=com.google.android.apps.secrets&hl=ru) or [YouTube](https://play.google.com/store/apps/details?id=com.google.android.youtube&hl=ru) 🤔

Reply:
I think that Playbook uses the same trick